src/IssueViz/Editor/SelectedIssueTagging/Adornment/IssueLocationAdornment.cs

```diff
@@ -28,37 +28,42 @@ namespace SonarLint.VisualStudio.IssueVisualization.Editor.SelectedIssueTagging.
     internal class IssueLocationAdornment : Button
     {
         private readonly TextBlock textBlock;
-        public IAnalysisIssueLocationVisualization Location { get; }
+        public IAnalysisIssueLocationVisualization LocationViz { get; }
 
-        public IssueLocationAdornment(ISelectedIssueLocationTag issueTag, IFormattedLineSource formattedLineSource)
+        public IssueLocationAdornment(IAnalysisIssueLocationVisualization locationViz, IFormattedLineSource formattedLineSource)
```

Comment:
Miscellaneous: it's simpler just to pass the location visualisation - we weren't using the tag.